src/interfaces/ws.js

```diff
@@ -166,6 +166,7 @@ module.exports = function(app) {
       }
 
       primus.on('connection', function(spark) {
+        console.log(spark.id)
```

Comment:
Oops?

Reply:
🤦 You do careful `git add -p` and then the reptile brain enters `git commit -am ...`